docs/index.html

```diff
@@ -14,9 +14,25 @@ <h2>Contents</h2>
 <li><a href="./virtual-dom/">Virtual DOM</a></li>
 <li><a href="./relational-db/">Relational DB</a></li>
 <li><a href="./precedence-climbing/">Precedence climbing parser</a></li>
-<li><a href="./just-in-time/">JIT</a></li>
+<li><a href="./just-in-time/">JIT</a>
 <li><a href="./bit-sliced-signatures/">Bit-sliced signatures</a></li>
+<li><a href="./json-parser/">JSON parser</a>
 </ul>
+<blockquote>
```

Comment:
This seems like an artifact of a botched Git merge.

Reply:
Good catch, not sure what happened.